codegen/src/main/java/software/amazon/awssdk/codegen/poet/builder/BaseClientBuilderClass.java

```diff
@@ -725,11 +726,19 @@ private MethodSpec invokePluginsMethod() {
                                                .addModifiers(PROTECTED)
                                                .addParameter(SdkClientConfiguration.class, "config")
                                                .returns(SdkClientConfiguration.class);
-        builder.addStatement("$T plugins = plugins()",
+        builder.addStatement("$T externalPlugins = plugins()",
+                             ParameterizedTypeName.get(List.class, SdkPlugin.class));
+
+        builder.addStatement("$T sdkPlugins = getSdkPlugins()",
```

Comment:
All plugins are SdkPlugins since that's the interface name. Can we differentiate the list of internal plugins by calling it `internalPlugins` instead of `sdkPlugins`?

Reply:
Done.